docs/tutorials/basic-usage.md

```diff
@@ -23,16 +23,16 @@ Basic usage of Shaka Player is very easy:
   <body>
     <video id="video"
            width="640"
-           poster="//shaka-player-demo.appspot.com/assets/poster.jpg"
-           controls autoplay></video>
+           poster="/assets/poster.jpg"
```

Comment:
The original version was correct.  assets/poster.jpg is a dynamic script hosted on appspot and does not live in the sources.  This change will cause the poster not to show in a local copy.  Please revert.

Reply:
You're right. I'll revert.